pkg/generate/app/pipeline.go

```diff
@@ -140,24 +141,24 @@ func (g PipelineGroup) String() string {
 	return strings.Join(s, "+")
 }
 
-const MaxServiceNameLen = 24
+const maxServiceNameLength = 24
```

Comment:
didn't seem like it: https://github.com/openshift/origin/blob/master/Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/api/generate.go#L60